pkg/sdk/groups_test.go

```diff
@@ -677,6 +681,9 @@ func TestUpdateGroup(t *testing.T) {
 
 	group.ID = generateUUID(t)
 
+	updatedDesc, err := nullable.ParseString(updatedDescription)
```

Comment:
Here also the same, Can we have another function in nullable which convert normal string to nullable string like below 

```go
func String(s string) (Value[string]) {
    return Value[string]{Set: True, Value: s}
}
```